projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -102,7 +100,14 @@ export class RuleSimpleConditionUiComponent implements OnInit, ControlValueAcces
     this.setDisabledState(isReadOnly);
   }
 
-  constructor(private config: AppConfigService, private categoryService: CategoryService) {
+  private readonly disabledTags = !this.tagService.areTagsEnabled();
+  private readonly disabledCategories = !this.categoryService.areCategoriesEnabled();
+
+  readonly fields = ruleConditionFields.filter(
+    (condition) => !((this.disabledTags && condition.name === 'tag') || (this.disabledCategories && condition.name === 'category'))
+  );
+
+  constructor(private config: AppConfigService, private categoryService: CategoryService, private tagService: TagService) {
```

Comment:
Pre-existing issue, but since the AppConfigService object is only being used inside the constructor itself, we should remove the 'private' keyword from the constructor signature for it.
```
  constructor(config: AppConfigService, private categoryService: CategoryService, private tagService: TagService) {
    this.mimeTypes = config.get<Array<MimeType>>('mimeTypes');
  }
```

Reply:
good catch, done